apps/keyboard/js/layouts/my.js

```diff
@@ -93,25 +107,27 @@ Keyboards.my = {
   }, {
     needsCommaKey: true,
     alt: {
-      '€': ['¢', '¥'],
+      '\'': ['‘', '’'],
```

Comment:
I don't know the requirement of symbols for Myanmar language, but if possible, please take en.js as an ref., we have done some modification to that based on UX input. Thanks.